managers/templates/managers/project_detail.html

```diff
@@ -42,7 +42,8 @@ <h2>
   <div class="container">
     <br/>
     {% for user in project.managers.all %}
-    <div class="col-sm-3">
+    <a href="{% url 'author-report-project-list' pk=project.pk user_pk=user.pk year=year month=month %}">
```

Comment:
Please indent correctly here and below in this template.

Reply:
done